lib/jsdom/browser/Window.js

```diff
@@ -77,32 +78,43 @@ function Window(options) {
   this.clearTimeout = stopTimer.bind(this, window);
   this.__stopAllTimers = stopAllTimers.bind(this, window);
 
+  var wrapConsoleMethod = function(method) {
+    return function() {
+      var args = Array.prototype.slice.call(arguments);
+      vConsole.fireEvent(window, method, args);
+    };
+  };
+
   this.console = {
```

Comment:
No need to keep the `raise`s. I'd like to get rid of `raise` entirely, actually.